content/docs/esc/administration/audit-logs.md

```diff
@@ -0,0 +1,73 @@
+---
+title_tag: Audit Logs | Pulumi ESC
+meta_desc: Pulumi ESC audit logs allow you to account for user activity within your organization.
+title: Audit Logs
+h1: Pulumi ESC audit logs
+meta_image: /images/docs/meta-images/docs-meta.png
+menu:
+    esc:
+        name: Audit Logs
+        parent: pulumi-esc-admin
+        weight: 1
+---
+
+{{% notes "info" %}}
+Audit Logs are available to organizations using the Enterprise and Business Critical editions.
+To learn more about editions, visit the [pricing page](/pricing/).
+{{% /notes %}}
+
+Audit logs enable you to track the activity of users within your ESC environments. Logs are immutable and record all user activity, providing critical visibility for security and compliance in your organization.
+
+ESC audit logs allow you to:
+
+- Monitor who accessed or modified secrets
+- Track environment creation and configuration changes
+- Ensure compliance with security policies
+- Provide attributable records for security forensics
+
+All ESC activities are recorded in Pulumi Cloud audit log system, capturing the timestamp, user identity, specific action taken, and source IP address for each event. You can download a CSV format or use Pulumi Cloud REST for exporting audit log events.
+
+## View Audit Logs
+
+To view audit logs as an organization admin:
+
+1. Navigate to the organization's **Settings** tab.
+1. Navigate to **Audit Logs** tab.
+
+This will show the most recent events in descending order. You can also filter logs by a particular user by selecting their profile picture.
+
+![View ESC audit logs in the ESC console](/docs/esc/assets/pulumi-view-audit-logs.png)
+
+## ESC audit log events
+
+| Event                                      | Description                                                                                                                                                       |
+|--------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Environment Created                      | indicates the creation of an environment                                                                         |
```

Comment:
these descriptions need work. at minimum, i would replace "indicates" with "records" in all of these.